modules/adding-new-extension-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="adding-new-extension-dynamic-plugin_{context}"]
+= Adding a new extension to your plug-in
+You can add different extensions to your plug-in that are loaded at runtime. Here is how you can add an example extension to your plug-in
+
+[id="adding-tab-to-pods-page"]
+== Adding a tab to the pods page
+You can add a tab to the *Pod Details* page.
+
+.Procedure
+
+. Add this extension to the `console-extensions.json` file.
++
+[source,json]
+
+----
+{
+    "type": "console.page/resource/tab",
+    "properties": {
+        "name": "Example Tab",
+        "href": "example",
+        "model": {
+            "group": "core",
+            "version": "v1",
+            "kind": "Pod"
+        },
+        "component": { "$codeRef": "ExampleTab" }
+}
+----
+
+. Edit the `package.json` file.
++
+[source,json]
+
+----
+        "exposedModules": {
+            "ExamplePage": "./components/ExamplePage",
+            "ExampleTab": "./components/ExampleTab"
+        }
+----
+
+. Create a new file `src/components/ExamplePage.tsx` to see your text when loading your plugin.
```

Comment:
the new file should be with name `src/components/ExampleTab.tsx`

Reply:
```suggestion
. Create a new file `src/components/ExamplePage.tsx` and add the following to see your text when loading your plug-in:
```
I don't have a suggestion for "to see your text...." Is the last part of the sentence important? If so, what if we said something like, "Write a message to display when your plug-in loads by creating a new file named _x_ and adding the following script:"

Reply:
It's text that displays on a new custom tab on the pods page (however we want to word that), or we could leave it out.